package-lock.json

```diff
@@ -2486,6 +2486,14 @@
                 "json-to-ast": "^2.0.3",
                 "jsonpointer": "^4.0.1",
                 "leven": "^3.1.0"
+            },
```

Comment:
There are still references to `json-pointer` in the package-lock. Not sure if that is perfectly normal or not in this type of situation. Did you `npm uninstall` the old `json-pointer` module?

Reply:
I did use `npm uninstall` let me have a look

Reply:
is this due to `json-pointer` being a [dependency ](https://github.com/CriminalInjuriesCompensationAuthority/data-capture-service/blob/f4ee58564d99e7a955d1ce089a9aea1ecec46a5f/package-lock.json#L9582) of `json-translator`?

Reply:
![image](https://user-images.githubusercontent.com/41323244/142193299-58b8d716-b44a-4413-b01d-57631f25a15f.png)
dependencies of redoc and openapi-sampler

Reply:
if it is due to it being a dep of `json-translator` then you would need to reinstall the translator on here because your latest version of the `json-translator` does not have `json-pointer` as a dependency.

I thing that makes sense :D

Reply:
I'm not fully understanding this. Happy to call you about it. Those listed deps shouldn't be there if they are listed as deps of the now-updated json-translator module. Unless I am misunderstanding the issue in the first place!